bbn-test-4/finality-providers/scripts/verify-new-fp-offchain.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash -eu
+
+# USAGE:
+# ./verify-new-fp-offchain.sh
+
+# checks for modified files in **/bbn-test-4/finality-providers/** compared to main branch
+# and validates if the finality provider registration has valid values to send the
+# transaction to BTC chain.
+
+CWD="$( cd -- "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )"
+. $CWD/fp-changed.sh
```

Comment:
What's this dot in the beginning?

Reply:
loading that file so that we can use the function `. $CWD/fp-changed.sh` and not duplicate the code 
But I could duplicate the code and remove this file :sweat_smile:

Reply:
I called that file utils and removed the one where it was just calling the two scripts to verify new fp